test/wanda_web/views/execution_view_test.exs

```diff
@@ -1,20 +1,20 @@
-defmodule WandaWeb.ListResultsViewTest do
+defmodule WandaWeb.ExecutionViewTest do
   use WandaWeb.ConnCase, async: true
 
   import Phoenix.View
   import Wanda.Factory
 
   alias Wanda.Results.ExecutionResult
 
-  describe "ResultView" do
+  describe "ExecutionView" do
```

Comment:
We could test this new logic about sending nil values in this conditions

Reply:
Added in the last commit :wink: